MrtCore/console_unpackaged_app/console_unpackaged_app.cpp

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include <windows.h>
+#include <iostream>
+
+#include "winrt\Windows.Foundation.h"
+#include "winrt\Windows.Foundation.Collections.h"
+#include "winrt\Microsoft.ApplicationModel.Resources.h"
+
+#include <MddBootstrap.h>
+
+using namespace winrt;
+using namespace winrt::Microsoft::ApplicationModel::Resources;
+
+HRESULT MddBootstrapInitializeWrapper()
+{
+    // Take a dependency on Project Reunion v0.8 preview.
+    const UINT32 majorMinorVersion{ 0x00000008 };
+    PCWSTR versionTag{ L"preview" };
+    const PACKAGE_VERSION minVersion{};
+    HRESULT hr{ MddBootstrapInitialize(majorMinorVersion, versionTag, minVersion) };
+    if (FAILED(hr))
+    {
+        wprintf(L"Error 0x%X in MddBootstrapInitialize(0x%08X, %s, %hu.%hu.%hu.%hu)\n",

```

Comment:
nit: maybe use `0x%08X` for the HR value too?

Reply:
for any hr you care about, "08" is implied :)